admin/securedrop_admin/__init__.py

```diff
@@ -403,6 +422,16 @@ def __init__(self, args):
              SiteConfig.ValidateLocales(self.args.app_path),
              string.split,
              lambda config: True],
+            ['v2_onion_services', self.check_for_v2_onion(), bool,
+             u'Do you want to enable v2 onion services?',
+             SiteConfig.ValidateYesNo(),
+             lambda x: x.lower() == 'yes',
+             lambda config: True],
+            ['v3_onion_services', self.check_for_v3_onion, bool,
```

Comment:
@redshiftzero This dynamic function checks if the user input for `v2_onion_services` was `no`, and  then `v3_onion_services` is must to `yes`. But, if `v2_onion_servies` was set to `yes`, and previously the admin set `v3_onion_service` to `no`, then it will show `no` as default value.